### PR TITLE
Improve documentation

### LINKED ISSUE #323: Improve find documentation
Re-evaluate whether this is necessary
Mention find doesn't affect assigned: #269 
Details: #310, #307

### LINKED ISSUE #327: Fix documentation errors
(Priority high to low)
Remove mention about list removing additional params: #317 
Duplicates: #252
Help command: #295
Typo: #313
UI component section not updated: #315
Incorrect MSS positioning: #316
Inconsistent command usage (find, etc. ): #255  
Extra glossary term (vendor rating): #263
Mention that exact same matching string for name would not be allowed 

(abit whack but why not)
Delete would fail for associated entities - DG: #257

### PR DESCRIPTION
**Pull request by kahsuann**
Resolves #323, #327, #328

#307: Clarify date format should follow displayed
#310: Operation will fail if no keyword provided
#267: Find command affects only assignable list and not assigned list when in view mode

#317: Remove mention about list removing additional params
#295: Refine `Getting Help` in DG
#313: en event -> an event in UG
#315: Person -> Vendor / Event in UI component
#316: MSS in new line
#255: Add empty space to find commands
#263: Removed vendor rating from glossary

Mention that same matching string for name would not be allowed in UG

#257: Add that delete would fail for associated entities under manual testing
#306: Give example for cumulative tagging
#293: Refine launching jar instructions
#291: not done yet, don't really agree with the bug

### CODE REVIEW

**Comment by Jasperabez** [comment]
standardize with spacing

File: docs/UserGuide.md [hunk 1]
```diff
@@ -151,13 +146,14 @@ Creates a vendor or event in EventTory.
...
-* To create en event: `create e/ n/EVENT_NAME on/DATE [t/TAG]…​`
+* To create an event: `create e/ n/EVENT_NAME on/DATE [t/TAG]…​`
 
 Note:
-If parameters are provided for the `v/` and `e/` flags, they will be ignored.
+* If parameters are provided for the `v/` and `e/` flags, they will be ignored.
 <box type="tip" seamless><br>
 Tip: A vendor or event can have any number of tags (including 0).
 </box>
+* If an event or a vendor already has the same name, command will fail and error message will be displayed.
```
Comment (Jasperabez) [inline docs/UserGuide.md:156]: might need to mention exact matching (case sensitive, white space etc.)

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -619,8 +620,6 @@ Preconditions: User is viewing an item.
 
 10. **Vendor:** A service provider, such as a caterer, photographer, decorator, who is employed for events.
 
-11. **Vendor Rating:** A qualitative score assigned to a vendor to track their past performance in events.
-
 12. **Item**: Refers to any entity or object that can be viewed or deleted within the system, such as an event, vendor.
 
 13. **Associated Entities**: Refers to any related items or dependencies connected to the main item. For example, for an event, associated entities include assigned vendors; for a vendor, associated entities include assigned events.
```
Comment (C5hives) [inline docs/DeveloperGuide.md:625]: Numbering broke due to deletion of definition #11

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -681,12 +682,12 @@ testers are expected to do more *exploratory* testing.
 1.  Delete a Vendor:
 
     1.  Type `delete v/1` <br>
-        Expected: The first vendor in the list is removed from EventTory.
+        Expected: The first vendor in the list is removed from EventTory if it does not have any assigned event. Else, deletion will fail.
```
Comment (C5hives) [inline docs/DeveloperGuide.md:685]: Might want to rephrase to: "if it is not assigned to any event. Else, the deletion will fail."

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -681,12 +682,12 @@ testers are expected to do more *exploratory* testing.
...
 
 2.  Delete an Event:
 
     1. Type `delete e/2` <br>
-       Expected: The second event in the list is removed from EventTory.
+       Expected: The second event in the list is removed from EventTory if it does not have any assigned vendor. Else, deletion will fail.
```
Comment (C5hives) [inline docs/DeveloperGuide.md:690]: Same as previous

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -763,7 +764,7 @@ testers are expected to do more *exploratory* testing.
 1. Getting Help:
 
    1. Type `help` in the command box and press Enter.
-      Expected: A help window appears, showing instructions and command summaries.
+      Expected: A help window appears with a URL directed to the user guide.
```
Comment (C5hives) [inline docs/DeveloperGuide.md:767]: "with a link to the user guide" might be more concise.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -151,13 +146,14 @@ Creates a vendor or event in EventTory.
...
-* To create en event: `create e/ n/EVENT_NAME on/DATE [t/TAG]…​`
+* To create an event: `create e/ n/EVENT_NAME on/DATE [t/TAG]…​`
 
 Note:
-If parameters are provided for the `v/` and `e/` flags, they will be ignored.
+* If parameters are provided for the `v/` and `e/` flags, they will be ignored.
 <box type="tip" seamless><br>
 Tip: A vendor or event can have any number of tags (including 0).
 </box>
+* If an event or a vendor already has the same name (exact matching - case sensitive), command will fail and error message will be displayed.
```
Comment (C5hives) [inline docs/UserGuide.md:156]: Consider changing to "**the** command will fail and **an** error message will be displayed."

File: docs/UserGuide.md [hunk 7]
```diff
@@ -291,7 +288,11 @@ Note:
 * All attributes of the `Vendor` or `Event` are searched, i.e. name, phone number, date, descriptions and tags.
 * Vendors and Events matching at least one keyword will be returned (i.e. `OR` search).
   * e.g. `party wedding` will return `Birthday Party`, `John's Wedding`
+* If user is viewing an event or a vendor, the `find` command will affect the assignable list only. Assigned list is not affected.
```
Comment (C5hives) [inline docs/UserGuide.md:291]: Do you think it's worth adding an image here to clarify between an "assignable list" and an "assigned list"?

File: docs/UserGuide.md [hunk 8]
```diff
@@ -291,7 +288,11 @@ Note:
...
+* To find an event according to its `date`, the displayed format should be followed.
```
Comment (C5hives) [inline docs/UserGuide.md:292]: Consider rephrasing to "To search the date field, the displayed format of dates should be followed"?

Should we add more information about why this is the case? Something like: "String matching is used to search date fields, so the keywords `November` and `Nov` will not match a date with month `11`.

**Review by C5hives — CHANGES_REQUESTED** [review]
Blocking @Jasperabez approval as there is an issue with numbering. Also left some extra suggestions (I hate how nitpicky the testers are)

**Review by C5hives — APPROVED** [review]
@kahsuann turns out the `findOnView.png` isn't very helpful (that's my bad), so it has been removed (thanks @Jasperabez).

The rest looks good!

**Pull request merged**